dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -125,7 +124,7 @@ public boolean processEntrySpecificCommands(List<String> args, String activeComm
     }
 
     private static void printToolList(List<DockstoreTool> containers) {
-        containers.sort(new ToolComparator());
+        containers.sort((c1, c2) -> c1.getPath().compareToIgnoreCase(c2.getPath()));
```

Comment:
Just curious, is this required for Java 11, or are you just streamlining the code?

Reply:
It's one of the comments in one of the commits. 
Basically, I need to avoid this issue 
https://github.com/powermock/powermock/issues/958